tests/ui/unnecessary_reserve.stderr

```diff
@@ -0,0 +1,52 @@
+error: unnecessary call to `reserve`
+  --> $DIR/unnecessary_reserve.rs:20:5
+   |
+LL |     vec.reserve(1);
+   |     -------------- help: remove this line
+LL |     vec.extend([1]);
+   |     ^^^^^^^^^^^^^^^^
+   |
+   = note: `-D clippy::unnecessary-reserve` implied by `-D warnings`
+
+error: unnecessary call to `reserve`
+  --> $DIR/unnecessary_reserve.rs:24:5
+   |
+LL |     vec.reserve(array.len());
+   |     ------------------------ help: remove this line
+LL |     vec.extend(array);
+   |     ^^^^^^^^^^^^^^^^^^
+
+error: unnecessary call to `reserve`
+  --> $DIR/unnecessary_reserve.rs:29:9
+   |
+LL |         vec.reserve(1);
+   |         -------------- help: remove this line
```

Comment:
I think the span of the suggestion is wrong? Shouldn't it include the `;` as well?